docs/topics/vscode-extension-reviewing-projects.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-reviewing-projects_{context}']
+= Reviewing issues
+
+You can use the {ProductShortName} extension icons to prioritize issues based on their severity. You can also see which issues have a a _Quick Fix_ automatic code replacement.
+
+.Procedure
+
+. Select a run configuration directory on the Run configuration pane and expand its folders to view the *Hints* generated for each application file.
+. Select a hint to view the source code.
+. Right-click a hint and select *View Details* to view the *Rule ID* and other information.
+. Prioritize issues based on the following icons, which are displayed next to each issue:
++
+.{ProductShortName} extension issue icons
+[cols="15%a,85%",options="header",]
+|====
+|Icon |Description
+|image::vs_mandatory.png[Mandatory] |The issue must be fixed for a successful migration.
+|image::vs_optional.png[Optional or Quick Fix] |The issue is  optional to fix for migration.
+|image::vs_potential.png[Warning] |The issue might be an issue during migration.
+|====
++
+You can easily identify Quick Fixes for issues: they are marked in the hints by the word *Quickfixes* and by the same icon that is used for optional issues.
```

Comment:
Do you need this sentence? I think it might be better to put it beside the "optional" icon.